templates/mock.html

```diff
@@ -0,0 +1,31 @@
+<div id="mock-section" class="section-profile">
+    <div class="row">
+         <div class="topleft duk-icon"><img onclick="removeSection('mock-section')" src="/gui/img/x.png"></div>
+         <div class="column" style="flex:40%;padding:15px;text-align: left">
+            <h1 style="font-size:70px;margin-top:-20px;">Mock</h1>
+            <h2 style="margin-top:-55px">simulated agents</h2>
+            <p>Mock adds a set of simulated agents to CALDERA and allows you to run complete operations without hooking any other computers up to your server. All simulated agents will appear as normal agents in the Chain plugin GUI and can be used in any operation.</p>
```

Comment:
The chain plugin doesn't exist anymore -- I'll make a ticket to update fieldmanual. We can probably skip the info about writing the `agents.yml` file, too. Maybe something like:

```
Mock adds a set of simulated agents to CALDERA and allows you to run complete operations without hooking any other computers up to your server. All simulated agents will appear as normal agents in the UI and can be used in any operation.
```

Reply:
Updated!